west.yml

```diff
@@ -19,7 +19,7 @@ manifest:
       path: modules/crypto/mbedtls
     - name: zephyr
       remote: zephyrproject-rtos
-      revision: a36c91b4cc49b9b54030121004a1565d6fc69475
+      revision: pull/85144/head
```

Comment:
Why not just update this to the latest upstream `main` branch, since the PR was already merged?

Reply:
I think you should likely also start pointing at the upstream hal_silabs tree, i.e. remove the reference to the downstream one (which will eventually be removed completely)

Reply:
> Why not just update this to the latest upstream `main` branch, since the PR was already merged?

I see you created this before the upstream PR was merged, which explains the reference. But now it can be updated to point at a real commit.

Reply:
> I think you should likely also start pointing at the upstream hal_silabs tree

Ideally yes, but for now:

```
zephyr/modules/hal/silabs$ git diff --stat upstream/main silabs/silabs
[...]
 113 files changed, 49889 insertions(+), 5172 deletions(-)
```
(the differences mainly come from mbedtls/psa).

Reply:
Oh, I didn't realize we had that in our downstream tree. That needs to be sorted out first then, i.e. in the short term rebase our hal_silabs on upstream and in the long term upstream any changes, so that we can proceed with the "remove public downstream hal_silabs" plan.